Xamarin.Essentials/FileSystem/FileSystem.ios.tvos.watchos.cs

```diff
@@ -50,7 +50,14 @@ internal static string PlatformGetContentType(string extension)
 
             var id = UTType.CreatePreferredIdentifier(UTType.TagClassFilenameExtension, extension, null);
             var mimeTypes = UTType.CopyAllTags(id, UTType.TagClassMIMEType);
-            return mimeTypes.Length > 0 ? mimeTypes[0] : null;
+            if (mimeTypes != null && mimeTypes.Length > 0)
```

Comment:
return (mimesTypes?.Length ?? 0) > 0 ? mimeTypes[0] : null;

Reply:
I’m old school readable code, like my whitespace :-] Will tweak in the am if no more suggestions

Reply:
You can also just do

return mimesTypes?.Length > 0 ? mimeTypes[0] : null;

Null never matches int, so will always be false.